### PR TITLE
add link to raw log

### PR DESCRIPTION
**Pull request by razanbilwani**

### CODE REVIEW

**Comment by razanbilwani** [comment]
@oba11 This...

**Comment by cirpo** [comment]
👍

File: src/client/components/buildOutput.jsx [hunk 1]
```diff
@@ -57,11 +57,11 @@ class BuildOutput extends React.Component {
   render() {
     return <div>
       <div className="build-output__header">
-        <span className="title">Build Id</span> : {this.props.buildId} <br/><br/>
+        <span className="title">Build Id</span> : <a href="{`/api/builds/${this.props.buildId}/log`}" target="_blank">{this.props.buildId}</a> <br/><br/>
```
Comment (p16) [inline src/client/components/buildOutput.jsx]: 👏

**Comment by oba11** [comment]
merging, cc @odino

**Pull request merged**